src/Exception/ContainerModificationsNotAllowedException.php

```diff
@@ -10,9 +12,24 @@
 
 use DomainException;
 
+use function sprintf;
+
 /**
  * @inheritDoc
  */
 class ContainerModificationsNotAllowedException extends DomainException implements ExceptionInterface
 {
+    /**
+     * @param string $service Name of service that already exists.
+     * @return self
+     */
+    public static function fromExistingService($service)
```

Comment:
Since this is a new method, and we're updating to PHP 7.4 as the minimum supported version, provide explicit argument and return typehints here, please.